_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,372 @@
+
+During the instance launch, the machine image on the instance performs some initial setup processes. These processes take a few minutes depending on the minutes, typically thought it should not take more than 5 mins. While this is happening your **Status Checks** reflect the **Initializing** status.
+
+
+## Task 2: Test your Gaia server
+
+Now, you are ready to test your Gaia server and make sure it is up and running.
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Choose **All services > EC2**.
+
+   The system displays the **EC2 Dashboard**.
+
+   ![](/storage/images/ec2-dashboard.png)    
+
+3. Select **Running Instances**.
+
+   The system displays your running instances.
+
+4. Locate your recently launched Gaia server.
+
+   Make sure the instance shows as running and **Status Checks** are complete.
+   Completed status checks ensures the Gaia processes and service were started.
+
+5. Select the **Description** tab.
+
+   ![](/storage/images/ec2-instance.png)   
+
+6. Locate the **IPv4 Public IP** value.
+7. Copy the IP and paste it in your browser.
+
+   You should see a message that your connection is not private.
+
+8. Press **Advanced**.
+
+   ![Hub test](/storage/images/private-connection.png)
+
+9. Choose to proceed.
+10. Extend the IP with the `PUBLIC_IP/hub_info` tag like so.
+
+    You should see a response from your Gaia hub!
+
+    ![Hub test](/storage/images/aws-hub.png)
+
+    At this point, you should see a **Not secure** message in the browser. That's because you haven't yet enabled SSL certification.
+
+
+## Task 3: Configure a domain name
+
+At this point, you can point a domain to your Gaia hub. Although it's not required, it is highly recommended. If you use a domain, you can migrate your instance to a different server (or even provider such as Azure or Dropbox) at any time, and still access it through the domain URL. Just point your domain at the IP address for your EC2 instance. Use an `A Record` DNS type.
+
+These instructions assume you have already created a free <a href="https://www.freenom.com" target="\_blank">domain through the freenom service</a>. To point this freenom domain to your Gaia hub, do the following:
+
+1. Log into your freenom account.
+2. Choose the **Manage Freenom Domain** tab.
+3. Add an **A** record leave the **Name** field blank.
+
+   This points your entire domain to the hub IP.
+
+4. Save your changes.
+
+5. Create a CNAME record.
+
+   For example, you can use the prefix `www` with your domain name. When you are done, your
+
+6. Save your changes.
+
+   At this point, your DNS management should look similar to the following except that with your domain rather than the `maryhub.ga` domain.
+
+   ![DNS fields](/storage/images/aws-dns-fields.png)
+
+7. After your changes propagate, visit your new domain at the `hub_info` page.
```

Comment:
For me -- this sent me to https://my-domain-name -- which triggered a "Your connection is not private" page. We might want to add a note here.

Reply:
Added a note this morning.